<launch>

    <node name="sp_ball_tracking" pkg="sp_perception" type="ball_tracker_node.py" output="screen">
        <remap from="input/image_raw" to="/camera/color/image_raw"/>
    </node>
</launch>